docs/UPLOAD.md

```diff
@@ -44,6 +44,42 @@ cp -X ~/Downloads/pico-os.uf2 /Volumes/RPI-RP2
 
 Open up your game in the editor, click the "Run on Device" button in the top right, and select the Sprig from the menu (again, like a USB device.)
 
-*Note: Make sure you're using a Chromium-based browser such as Edge or Chrome. On Linux you will also need to add your user to the `dialout` group*
-
 That's it! You're done :)
+
+> [!NOTE]
+> Make sure you're using a Chromium-based browser such as Edge or Chrome.
+> If you're using Linux, check the Linux section below because you probably need more setup!
+
+
+## But I use Linux!
+
+Chances are, you need to do some additional setup to flash your game on Linux. But don't fear - it's straightforward!
+
+### Join the group!
+
+For most Linux distributions, you need to add your user to the `dialout` group. If this group exists, or it doesn't work, add your user to the `uucp` group instead.
```

Comment:
Are you sure you didn't mean "If this group doesn't exist"? Lol this PR showed up in my GitHub feed this morning and that just didn't really sound right

Reply:
Nice catch!

Reply:
Resolved with #2200